lib/rake-terraform/default_tasks.rb

```diff
@@ -9,10 +10,16 @@
   # Set to string 'false' instead of bool so users can more-easily override
   hide_tasks = ENV['TERRAFORM_HIDE_TASKS'] || 'false'
 
+  # Might need a more comprehenvise regex to cover all cases
+  env_root = env_glob.match(%r{^((\w|\s|-)+\/?)+}).to_s
```

Comment:
Given that regex is pretty [write-only](https://en.wikipedia.org/wiki/Write-only_language), would you mind adding a comment with a sample input/output here?